modules/rosa-policy-shared-responsibility.adoc

```diff
@@ -96,8 +96,8 @@ Red Hat is responsible for enabling changes to the cluster infrastructure and se
 |===
 
 [id="rosa-policy-identity-access-management_{context}"]
-== Identity and access management
-The Identity and Access Management matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
+== Access and identity authorization
+The Access and Identity Authorization matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
```

Comment:
The capitalization of "Access and Identity Authorization" should be set to lowercase.